src/Button/SimpleButton/SimpleButton.spec.jsx

```diff
@@ -40,17 +40,6 @@ describe('<SimpleButton />', () => {
     expect(wrapper.find('button', {disabled: true}).length).toBe(1);
   });
 
-  it('warns if no click callback method is given', () => {
-    const wrapper = TestUtil.mountComponent(SimpleButton);
-    const logSpy = sinon.spy(Logger, 'debug');
-
-    wrapper.find('button').simulate('click');
-
-    expect(logSpy).toHaveProperty('callCount', 1);
-
-    Logger.debug.restore();
-  });
-
   it('calls a given click callback method onClick', () => {
     const onClick = sinon.spy();
```

Comment:
just minor and probably not for now: as we introduced and use `jest` in some other classes, it would be cool to replace `sinon` here as well..